source/OptimaJet.DWKit.StarterApplication/Templates/Notification.cshtml

```diff
@@ -0,0 +1,5 @@
+﻿<html>
+<body>
+    <h2> @Model.Message </h2>

```

Comment:
Eventually we will need better designed notification messages?

Reply:
Well, we said one template for notifications.  Most of "message design" will be in the message rather than the template.   I wasn't planning on doing more for this part in the near term.  If you have any suggestions, let me know.